packages/cache/src/CacheStore.spec.tsx

```diff
@@ -32,98 +32,137 @@ describe('CacheStore', () => {
+      } catch (promiseToSuspense) {
+        expect(await promiseToSuspense).toBeUndefined()
+      }
+      try {
+        cacheStore.suspend(errorCache(2))
+      } catch (error) {
+        expect(error).toBe(ERROR_MESSAGE)
+      }
+      expect(cacheStore.getError(errorCache(1))).toBe(ERROR_MESSAGE)
+      expect(cacheStore.getError(errorCache(2))).toBe(ERROR_MESSAGE)
 
-  it("have clearError method with key should clear promise & error for key's cached", async () => {
-    expect(cacheStore.getError(errorCache(1))).toBeUndefined()
-    expect(cacheStore.getError(errorCache(2))).toBeUndefined()
-    try {
-      cacheStore.suspend(errorCache(1))
-    } catch (promiseToSuspense) {
-      expect(await promiseToSuspense).toBeUndefined()
-    }
-    try {
-      cacheStore.suspend(errorCache(1))
-    } catch (error) {
-      expect(error).toBe(ERROR_MESSAGE)
-    }
-    try {
-      cacheStore.suspend(errorCache(2))
-    } catch (promiseToSuspense) {
-      expect(await promiseToSuspense).toBeUndefined()
-    }
-    try {
-      cacheStore.suspend(errorCache(2))
-    } catch (error) {
-      expect(error).toBe(ERROR_MESSAGE)
-    }
-    expect(cacheStore.getError(errorCache(1))).toBe(ERROR_MESSAGE)
-    expect(cacheStore.getError(errorCache(2))).toBe(ERROR_MESSAGE)
+      cacheStore.clearError(errorCache(1))
+      expect(cacheStore.getError(errorCache(1))).toBeUndefined()
+      expect(cacheStore.getError(errorCache(2))).toBe(ERROR_MESSAGE)
+      cacheStore.clearError(errorCache(2))
+      expect(cacheStore.getError(errorCache(1))).toBeUndefined()
+      expect(cacheStore.getError(errorCache(2))).toBeUndefined()
+    })
+  })
 
-    cacheStore.clearError(errorCache(1))
-    expect(cacheStore.getError(errorCache(1))).toBeUndefined()
-    expect(cacheStore.getError(errorCache(2))).toBe(ERROR_MESSAGE)
-    cacheStore.clearError(errorCache(2))
-    expect(cacheStore.getError(errorCache(1))).toBeUndefined()
-    expect(cacheStore.getError(errorCache(2))).toBeUndefined()
+  describe('remove', () => {
+    it('should remove specific cached', async () => {
+      try {
+        cacheStore.suspend(successCache(1))
+      } catch (promiseToSuspense) {
+        await promiseToSuspense
+      }
+      expect(cacheStore.getData(successCache(1))).toBe(TEXT)
+      expect(() => {
+        cacheStore.remove(successCache(1))
+      }).not.throw()
+      expect(cacheStore.getData(successCache(1))).toBeUndefined()
+    })
   })
 
-  it('should take a key and remove the cached data for that key', async () => {
-    try {
-      cacheStore.suspend(successCache(1))
-    } catch (promiseToSuspense) {
-      await promiseToSuspense
-    }
-    expect(cacheStore.getData(successCache(1))).toBe(TEXT)
-    expect(() => {
-      cacheStore.remove(successCache(1))
-    }).not.throw()
-    expect(cacheStore.getData(successCache(1))).toBeUndefined()
+  describe('reset', () => {
+    it('should reset data of all cached', async () => {
```

Comment:
```suggestion
    it('should delete all cached and notify to subscribers', async () => {
```

Add test code to check "reset" do notifying to subscribers after deleting all cached